docs/index.md

```diff
@@ -1,17 +1,84 @@
-Autonolas Contribute is a service that streamlines the contribution experience for members of the Autonolas community.
+![ContributionKit](images/autonolas-contribute-logo.png){ align=left width="150" }
 
-One of the key features of Autonolas Contribute is the ability to show off members' contributions with badges. When someone mints a badge, they will start at the first tier. As they complete actions that contribute to the success of Autonolas, you'll earn points and climb the leaderboard. When you earn enough points to reach a higher badge tier, their badge will automatically update to reflect the new rank. This is a great way to demonstrate contributions on NFT-enabled social media sites and earn recognition that reflects your contribution within the Autonolas community.
+The ContributionKit helps you build services to show off community contributions by minting badges which evolve as users make contributions to the DAO. Such services work by minting badges and activate them to witness an update, showing off contributions on NFT-enabled social media sites.
 
-Autonolas Contribute is powered by an autonomous service. We'll soon be releasing the code as an open-source toolkit. This will allow other teams to fork and run their own versions of the service, making it even easier for teams to track and manage contributions.
+Autonolas Contribute, a service based on the ContributionKit, streamlines the contribution experience for members of the Autonolas community.
+When someone mints a badge, they will start at the first tier. As they complete actions that contribute to the success of Autonolas, you'll earn points and climb the leaderboard. When you earn enough points to reach a higher badge tier, their badge will automatically update to reflect the new rank. This is a great way to demonstrate contributions on NFT-enabled social media sites and earn recognition that reflects your contribution within the Autonolas community.
```

Comment:
As they complete actions....THEY'll ear....

Reply:
Also in the next sentece, first person and second person mismatch.

Reply:
Fixed.